pkg/universe.dagger.io/x/ezequiel@foncubierta.com/terraform/plan.cue

```diff
@@ -0,0 +1,11 @@
+package terraform
+
+// File to write the output plan
+_#defaultPlanFile: string | *"./out.tfplan"
```

Comment:
Should be directly the type of `planFile`, you do not need an intermediate definition for that